stable/insights-agent/templates/nova/cronjob.yaml

```diff
@@ -27,6 +27,8 @@ spec:
             command:
               - /nova
               - find
+              - -- helm
```

Comment:
@sudermanjr, @rbren - should these flags be added via config file? Would the below work?
```
nova:
  ...
  config:
    context: ""
    desired-versions: {}
    include-all: true
    wide: true
    output-file: "/output/nova.json"
    helm: true
    containers: true
```

Reply:
I'm not sure on this one...I think flags are fine though.

Reply:
Flags are probably better for these, since we don't really want people to override them.

Reply:
I wonder if `.Config.configLocation` will not override those flags